README.md

```diff
@@ -237,6 +237,12 @@ Research papers and similar documents studying something related to Matrix.
   `Apache-2.0` `TypeScript`
 - [matrix-widget-debug](https://github.com/turt2live/matrix-widget-debug) - Test
   widget for validating clients and `matrix-widget-api`. `Apache-2.0` `JavaScript`
+- [AlexGames](https://github.com/alexbarry/AlexGames) - A simple board games
+  web app that works as a Matrix widget, see
+  ["This Week in Matrix" 2024-09-27](https://matrix.org/blog/2024/09/27/this-week-in-matrix-2024-09-27/#alexgames-simple-lua-rust-webassembly-powered-board-games-running-in-a-matrix-widget)
```

Comment:
Hmm, I would think most of the info from that TWIM post would also be available at a homepage or repo, and that does appear to be the case with this repo. The TWIM post then feels a bit redundant, so let's remove that.